app/src/main/java/bubbleheads/buy_cook/Recipe.java

```diff
@@ -8,14 +8,17 @@
     private final String recipeDetail;
     private final int recipePhoto;
     private final Category category;
```

Comment:
actually you can now drop Category from here. So you'll have list of Recipes and list of Categories in your app. It would be enough for you to use id in order to connect them